apps/emqx_gateway/src/bhvrs/emqx_gateway_conn.erl

```diff
@@ -700,6 +700,16 @@ parse_incoming(
 
 parse_incoming(<<>>, Packets, State) ->
     {Packets, State};
+parse_incoming(
```

Comment:
Since emqx_gateway_conn is generic code for all gateways, we should not handle stomp-related frame here.

It's better to handle it in emqx_stomp_frame.erl module

Reply:
OK, let me try

Reply:
IMO, you can return a new frame type, i.e. heartbeat and handle it in the channel module. But I am not sure if the heartbeat message contains only one `\n` char.🤔

Reply:
I have updated it, if it is not perfect, maybe you can improve it.